flash/text/question_answering/model.py

```diff
@@ -322,7 +321,7 @@ def _initialize_model_specific_parameters(self):
 
     def modules_to_freeze(self) -> Union[Module, Iterable[Union[Module, Iterable]]]:
         """Return the module attributes of the model to be frozen."""
-        return _get_question_answering_bacbones_for_freezing(self.model)
+        return self.model.base_model
```

Comment:
Is there a chance that there is no attribute `base_model` and this will error out?

Reply:
Awesome, thanks Ethan!